src/Catalogue.tsx

```diff
@@ -16,14 +16,33 @@ interface CatalogueProps {
 const Catalogue: React.FC<CatalogueProps> = ({schema}) => {
     const [rowData, setRowData] = useState<any[]>([]);
     const [columnDefs, setColumnDefs] = useState<ColDef[]>([]);
+    const fieldConfMap = {};
+
+    //TODO: remove this as it si not required
+    useEffect(() => {
+        Object.entries(data).map(([key, config]: [string, any]) => {
+            const fieldConfvalues = {};
+            Object.entries(config).map(([prop, value]: [string, any]) => {
+                // @ts-ignore
```

Comment:
What are these ts-ignore comments about?

Reply:
it complains about json is any type.